components/startadojo/Hero.vue

```diff
@@ -3,19 +3,28 @@
     <h1 class="c-hero__header1">{{ $t('Let\'s start a Dojo to help engage young people near you') }}</h1>
     <div class="c-hero__step-cards">
       <div class="c-hero__step-card">
-        <img class="c-hero__image" src="~/static/images/doc.svg" />
-        <h1 class="c-hero__step-header c-hero__orange">{{ $t('Step 1') }}</h1>
-        <p class="c-hero__step-text"><span class="c-hero__step-text-first-word c-hero__orange">{{ $t('Register ') }}</span>{{ $t('with us as a Champion so we can support you as you build your club.') }}</p>
+        <img class="c-hero__step-image" src="~/static/images/doc.svg" />
+        <h1 class="c-hero__step-header">
+          <span class="c-hero__step-number">{{ $t('1') }}</span>
+          <span class="c-hero__step-name">{{ $t('Register') }}</span>
```

Comment:
Still doesn't work great for LTR, but considering nothing as been prepared for that, it can prob wait